### DISCUSSION TITLE
Video Js Buffering Issue

### DISCUSSION THREAD

**Discussion by aravihipoint** [videojs/video.js#8280] [Q&A]
I have integrated video js for playing files stored in AWS S3 bucket so generally video url will be an s3 based url (.mp4 type video file). Now the issue is when i take the video its taking much time to buffer and play the video. I have enabled Transfer Acceleration attribute in the corresponding bucket as well. For safari in mac ,  media buffering time much higher than chrome in windows .  When i checked in browser console i could see multiple request are going to AWS  based on bytes . 

So anyone faced this issue before ? please suggest a solution 

Thanks

**Comment by lalit-tudip** [comment]
@aravihipoint Did you get any solutions or workaround? I'm also facing the same issue with videojs on Mac Safari.

**Comment by miguel-guzman84** [comment]
I have the same issue but, in my case, I'm testing locally the test and also having in a $web container in Azure and I'm having the same issue with the buffering.

**Comment by aravihipoint** [comment]
Hello

I have added accelerate keyword along with s3 urls thus solved my issue.
Example : https://{bucketname}.s3-accelerate.amazonaws.com
